app/src/main/java/at/roteskreuz/stopcorona/screens/debug/exposure_notifications/DebugExposureNotificationsViewModel.kt

```diff
@@ -33,29 +48,27 @@ class DebugExposureNotificationsViewModel(
         exposureNotificationClient.isEnabled()
             .addOnSuccessListener { enabled: Boolean ->
                 exposureNotificationsEnabledSubject.onNext(enabled)
-                exposureNotificationsErrorSubject.onNext("")
             }
             .addOnFailureListener { exception: Exception? ->
                 Timber.e(
                     exception,
                     "could not get the current state of the exposure notifications SDK"
                 )
-                //TODO: how do we handle this???
                 exposureNotificationsEnabledSubject.onNext(false)
-                exposureNotificationsErrorSubject.onNext("could not get the current state of the exposure notifications SDK: '${exception}'")
+                exposureNotificationsTextSubject.onNext("could not get the current state of the exposure notifications SDK: '${exception}'")
             }
     }
 
     fun observeEnabledState(): Observable<Boolean> {
         return exposureNotificationsEnabledSubject
     }
 
-    fun observeResolutionError(): Observable<DataState<Status>> {
+    fun observeResolutionError(): Observable<DataState<Pair<Status,DebugAction>>> {
```

Comment:
Suggestion: I think in this case the `DebugAction` can be a bit misleading, I would rename it to something like `ResolveResolutionError` or something that implies that an automatic action will be taken to resolve the error.